rosbag2_storage/test/rosbag2_storage/test_plugin.cpp

```diff
@@ -79,6 +80,12 @@ rosbag2_storage::BagMetadata TestPlugin::get_metadata()
   return rosbag2_storage::BagMetadata();
 }
 
+std::string TestPlugin::get_relative_path() const
+{
+  std::cout << "\nreturning relative path\n";
+  return plugin_constants::DUMMY_FILEPATH;
```

Comment:
I can't comment on the lines above, but consider shifting the return value `default_max_bagfile_size` into the `plugin_constants` header as well.